parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -591,9 +609,17 @@ public static Statistics toParquetStatistics(
     if (!stats.isEmpty() && stats.isSmallerThan(MAX_STATS_SIZE)) {
```

Comment:
So, we are not truncating if the size is too large? It should be the other way around: check the size after the truncate.

Reply:
Got it.

Reply:
fixed